search/sorting.py

```diff
@@ -14,8 +14,9 @@ class LoreSortingFields(object):
     SORT_BY_NR_ATTEMPTS = ('nr_attempts', 'Number of Attempts (desc)', '-')
     SORT_BY_AVG_GRADE = ('avg_grade', 'Average Grade (desc)', '-')
     SORT_BY_TITLE = ('titlesort', 'Title (asc)', '')
+    SORT_BY_RELEVANCE = ('_score', 'Relevance (asc)', '')
```

Comment:
It is the name of the special field in the ES Index: see https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-sort.html